py-utils/src/utils/msg_bus/message_bus/adaptee/confluent_kafka_adaptor.py

```diff
@@ -0,0 +1,109 @@
+import logging
+import sys
+from collections import namedtuple
+
+from confluent_kafka.cimpl import Producer, Consumer
+from confluent_kafka.admin import AdminClient, NewTopic, ClusterMetadata
+from message_bus.adaptee import Adaptee
+from message_bus.utils import log_decorator
+
+ConsumerRecord = namedtuple("ConsumerRecord",
+                            ["topic", "message", "partition", "offset", "key"])
+
+
+class ConfluentAdaptee(Adaptee):
+
+    def __init__(self, config):
+        try:
+            self.config = config

```

Comment:
Please names like _config or __config for all the protected or private members as conventions.